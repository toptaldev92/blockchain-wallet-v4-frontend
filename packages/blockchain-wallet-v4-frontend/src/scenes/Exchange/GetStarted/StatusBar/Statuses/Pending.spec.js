import { Pending } from './Pending'
import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

jest.mock('blockchain-info-components', () => ({
  Text: 'text'
}))

describe('Pending', () => {
  it('renders correctly', () => {
    const component = shallow(<Pending />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
