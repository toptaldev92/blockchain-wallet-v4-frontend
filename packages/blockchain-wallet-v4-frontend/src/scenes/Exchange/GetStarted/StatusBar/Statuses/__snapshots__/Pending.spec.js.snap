// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pending renders correctly 1`] = `
<styled.div>
  <text
    color="white"
    size="14px"
    weight={500}
  >
    <FormattedMessage
      defaultMessage="Account Verification In Progress"
      id="scenes.exchange.getstarted.status.pending.title1"
      values={Object {}}
    />
  </text>
  <text
    size="14px"
    weight={400}
  >
    <FormattedMessage
      defaultMessage="We are currently reviewing your application. Hang tight! In just a few minutes you will be all set to trade cryptocurrency. You should receive an update within 5 minutes."
      id="scenes.exchange.getstarted.status.pending.description"
      values={Object {}}
    />
  </text>
</styled.div>
`;
